LondonDataServices.IDecide.Manage.Server.Tests.Acceptance/Models/PatientDecisions/Decision.cs

```diff
@@ -21,6 +22,13 @@ public class Decision
         public string ResponsiblePersonGivenName { get; set; }
         public string ResponsiblePersonSurname { get; set; }
         public string ResponsiblePersonRelationship { get; set; }
+
```

Comment:
Not sure why this model exist.  We alreadty have it under Models\Decision

Reply:
It existed before because there was a difference between what was expected and what was posted but I guess you're right they're the same now and can be removed